magento/magento2
# Direct STDERR output when listing crontab to /dev/null

### Description (*)
When installing crontab on systems using fcron the output from STDERR is captured and inserted into the updated crontab. An example from a fresh install follows. While fcron does not outright complain about this when saving the file, it is messy and quite avoidable.

```
2019-02-03 13:18:18  INFO listing magento's fcrontab
#~ MAGENTO START 7cd0a74c421051ec6cce8936eb9edd02
* * * * * /usr/lib64/php7.2/bin/php /var/www/magento/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/magento/var/log/magento.cron.log
* * * * * /usr/lib64/php7.2/bin/php /var/www/magento/update/cron.php >> /var/www/magento/var/log/update.cron.log
* * * * * /usr/lib64/php7.2/bin/php /var/www/magento/bin/magento setup:cron:run >> /var/www/magento/var/log/setup.cron.log
#~ MAGENTO END 7cd0a74c421051ec6cce8936eb9edd02
```

### Fixed Issues (if relevant)

N/A
### Manual testing scenarios (*)

1. Use fcron as system cron daemon
2. Execute **cron:install** command
3. View crontab and see it littered with STDERR output

### Contribution checklist (*)
